### PR TITLE
[Part 1] Update and move documentation from website to GitHub Wiki

### PR DESCRIPTION
**Pull request by Margen67**
This PR is in multiple parts due to committing to multiple branches, gh-pages and master.

* Wiki can be edited without waiting, and is easier to edit/preview.

Page changes:
* Sidebar: Add Discord link, make all links except Download redirects.
  * Moved Quickstart out of Development since it's a guide for users now.
  * Changed the icon of Quickstart and FAQ
* Gallery: Replace ancient videos with playlist of uploads from the Xenia YouTube channel.
* Compatibility: Replaced with redirect to wiki page to preserve old link.
* Download: Lower minimum Windows requirement to 7+, add Windows 10 requirement to d3d12

Old pages have been made into redirects so outdated links aren't broken.

Part 1 of #1392.

Preview: https://margen67.github.io/xenia/

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: About
+icon: info
+permalink: /about/
+---
+
+<meta http-equiv="Refresh" content=	"0; url=https://github.com/xenia-project/xenia/wiki/about">
```
Comment (gibbed) [inline about.html]: Even if you try to send the user to another URL this way, there's no guarantee they have this enabled. There should be content on the page noting the page move.

File: _config.yml [hunk 2]
```diff
@@ -50,8 +61,8 @@ navigation:
 #      icon: people
   - title: Development
     pages:
-    - title: Quickstart
-      url: https://github.com/xenia-project/xenia#quickstart
+    - title: Building
+      url: https://github.com/xenia-project/xenia/blob/master/docs/building.md
```
Comment (gibbed) [inline _config.yml]: Why is Building replacing Quickstart here?

File: about.md [hunk 3]
```diff
@@ -1,46 +0,0 @@
...
-None! There's a [roadmap](/roadmap/), but the project has largely achieved what
-I set out for it to do (waste time building something cool), and now is purely a
-passion project in my spare time. In addition to getting some of my favorite
-games to play I'm really just hacking to have fun.
-
-## Team
-
-Contributors can be seen [here on GitHub](https://github.com/xenia-project/xenia/graphs/contributors).
-
-TODO: auto generate a list from API.
```
Comment (gibbed) [inline about.md]: Why wikify the about page?

File: compatibility.html [hunk 4]
```diff
@@ -1,218 +1,7 @@
...
-      }
-
-      var query = this.value;
-      lastKeyTimerId = setTimeout(function() {
-        issuesTable.search(query).draw();
-      }, 100);
-    });
-  }
-</script>
+<meta http-equiv="Refresh" content="0; url=https://github.com/xenia-project/game-compatibility/issues">
```
Comment (gibbed) [inline compatibility.html]: See comment on `about.html`.

File: faq.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: FAQ
+icon: assignment
+permalink: /faq/
+---
+
+<meta http-equiv="Refresh" content=	"0; url=https://github.com/xenia-project/xenia/wiki/FAQ">
```
Comment (gibbed) [inline faq.html]: See comment on `about.html`.

File: gallery.html [hunk 6]
```diff
@@ -6,11 +6,5 @@
...
 
 <p>
-  <iframe height="480" src="https://www.youtube.com/embed/vE32Z9ytbgU?rel=0" frameborder="0" allowfullscreen></iframe>
-</p>
-<p>
-  <iframe height="480" src="https://www.youtube.com/embed/oAk0_jidpA0?rel=0" frameborder="0" allowfullscreen></iframe>
-</p>
-<p>
-  <iframe height="480" src="https://www.youtube.com/embed/TY5QeQ90iNs?rel=0" frameborder="0" allowfullscreen></iframe>
+  <iframe height="480"src="https://www.youtube.com/embed/videoseries?list=UU6bJhkUUHE18HVG2mpbygSQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (gibbed) [inline gallery.html]: Missing space here before `src`.

File: roadmap.html [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
+title: Roadmap
+icon: event
+permalink: /roadmap/
+---
+
+<meta http-equiv="Refresh" content=	"0; url=https://github.com/xenia-project/xenia/wiki/roadmap">
```
Comment (gibbed) [inline roadmap.html]: See comment on `about.html`.

**Comment by gibbed** [comment]
Additionally, `miscellaneous website update` in the commit message is not really acceptable here. Please make that more descriptive.

File: about.md (hunk 3 above)
Comment (Margen67) [reply about.md]: For consistency. I can undo it, though.

File: _config.yml (hunk 2 above)
Comment (Margen67) [reply _config.yml]: Quickstart was made into a guide on how to use Xenia and was a duplicate of building.md

File: roadmap.html (hunk 7 above)
Comment (illusion98) [reply roadmap.html]: Added a note about redirecting page.

File: gallery.html (hunk 6 above)
Comment (illusion98) [reply gallery.html]: Resolved.

File: faq.html (hunk 5 above)
Comment (illusion98) [reply faq.html]: Added a note about redirecting page.

File: compatibility.html (hunk 4 above)
Comment (illusion98) [reply compatibility.html]: Added a note about redirecting page.

File: about.html (hunk 1 above)
Comment (illusion98) [reply about.html]: Added a note about redirecting page.

**Comment by illusion98** [comment]
> 
> 
> Additionally, `miscellaneous website update` in the commit message is not really acceptable here. Please make that more descriptive.

Changed to `Update and move documentation from website to GitHub Wiki`

**Comment by illusion98** [comment]
Added Quickstart guide to sidebar and changed icon of FAQ

File: about.md (hunk 3 above)
Comment (gibbed) [reply about.md]: I would prefer this to be undone.

**Comment by gibbed** [comment]
If the two outstanding issues can be resolved (undoing about change and resolving merge conflict in `_config.yml`) this can be merged.

**Comment by gibbed** [comment]
Whoops. I didn't mean to approve the changes already. About change is still needed.

File: about.md (hunk 3 above)
Comment (illusion98) [reply about.md]: Undone.

**Pull request merged**